cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -639,13 +638,6 @@ Slice::Python::CodeVisitor::visitClassDefStart(const ClassDefPtr& p)
 
     _out.dec();
 
-    if (_classHistory.count(scoped) == 0 && p->canBeCyclic())
-    {
-        //
-        // Emit a forward declaration for the class in case a data member refers to this type.
-        //
-        _out << sp << nl << "_M_" << type << " = IcePy.declareValue('" << scoped << "')";
-    }
```

Comment:
Impossible to hit.

`visitClassDecl` also has this logic, and at the end, adds `scoped` to `_classHistory`.
Now that we're guaranteed to call `visitClassDecl` before `visitClassDef`,
`_classHistory.count(scoped) == 0` will always be false.

`visitInterfaceDecl` also has this logic. And since we're guaranteed to call `visitInterfaceDecl` before a call to `visitInterfaceDef`, `_classHistory` will always contain this element by this point.